systemd/org.nemomobile.devicelock.conf

```diff
@@ -2,8 +2,10 @@
 <busconfig>
   <policy user="root">
     <allow own="org.nemomobile.devicelock" />
+    <allow send_interface="org.nemomobile.devicelock.client.Authenticator"/>
   </policy>
   <policy context="default">
     <allow send_destination="org.nemomobile.devicelock" />
+    <allow receive_interface="org.nemomobile.devicelock.client.Authenticator"/>
```

Comment:
Hmm. Is one of these wrong? It says `client` here but it doesn't say that in https://github.com/sailfishos/sailjail-permissions/pull/101.